### PR TITLE
Feature update 2.9.0

### PR DESCRIPTION
**Pull request by Blackbaud-BrandonJones**

### CODE REVIEW

File: src/app/content/footer/index.html [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+        <stache-code>copyrightLabel</stache-code> — This flag allows you to specify the additional copyright information after the current date. Defaults to <stache-code>Blackbaud, Inc. All rights reserved.</stache-code>
+      </p>
+      <sky-alert
+      alertType="warning">
+        To use the default, your project must include a <stache-code>src/assets/locales/resources_en_US.json</stache-code> file.
+      </sky-alert>
+    </li>
+    <li>
+      <p>
+        <stache-code>nav</stache-code> — This flag contains an <stache-code>items</stache-code> array, allowing you to specify links to be included in the footer.
```
Comment (Blackbaud-DeniseCisneros) [inline src/app/content/footer/index.html]: This flag contains an <stache-code>items</stache-code> array, allowing you to include specific links in the footer.

File: src/app/content/footer/index.html [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+    </li>
+    <li>
+      <p>
+        <stache-code>siteName</stache-code> — Value read from the <stache-code>skyux.app.title</stache-code> attribute in your <stache-code>skyuxconfig.json</stache-code> file.
+      </p>
+    </li>
+  </ul>
+
+  <p>
+    To enable the footer, add <stache-code>footer</stache-code> to your <stache-code>skyuxconfig.json</stache-code> under <stache-code>appSettings: stache</stache-code>. This enables the footer globally across your SPA. To disable the footer on specific pages, add the `showFooter: false` attribute to the <a stacheRouterLink="/content/wrapper">stache wrapper</a>.
```
Comment (Blackbaud-DeniseCisneros) [inline src/app/content/footer/index.html]: To enable the footer globally across your SPA, add <stache-code>footer</stache-code> to your <stache-code>skyuxconfig.json</stache-code> under <stache-code>appSettings: stache</stache-code>. To disable the footer on specific pages, add the `showFooter: false` attribute to the <a stacheRouterLink="/content/wrapper">stache wrapper</a>.

File: src/app/support/changelog/2018-03/index.html [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+  showTableOfContents="true">
+
+  <stache-page-summary>
+    Monitor this page to keep up with the latest changes and releases for {{ stache.jsonData.global.productNameLong }}.
+  </stache-page-summary>
+
+  <stache-markdown>
+## 2.5.0 (2018-3-28)
+
+- Routing and navigation updates: [#426](https://github.com/blackbaud/stache2/pull/426)
```
Comment (Blackbaud-DeniseCisneros) [inline src/app/support/changelog/2018-03/index.html]: Capitalize all instances of URL for consistency, please :)

File: src/app/support/changelog/2018-03/index.html [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+  - Fixed a bug with breadcrumbs displaying the route params when present in the url.
+  - Fixed a bug with in-page navigation and fragment routing when route params were present in the url.
+  </stache-markdown>
+
+  <stache-page-anchor>
+    Stache Documentation Update (2018-03-07)
+  </stache-page-anchor>
+
+  <stache-markdown>
+- For Blackbaud staff, we created a new template designed for you. Thanks to Bobby Earl for the work he did to update the [SKY UX CLI (1.2.0) <i class="fa fa-external-link" aria-hidden="true"></i>](https://github.com/blackbaud/skyux-cli/blob/master/CHANGELOG.md) to support the internal template Git URL.
```
Comment (Blackbaud-DeniseCisneros) [inline src/app/support/changelog/2018-03/index.html]: For Blackbaud staff, we created a new template for you. Thanks to Bobby Earl for his work to update the [SKY UX CLI (1.2.0) <i class="fa fa-external-link" aria-hidden="true"></i>](https://github.com/blackbaud/skyux-cli/blob/master/CHANGELOG.md) to support the internal template Git URL.

File: src/app/support/changelog/2018-03/index.html [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+- To use the new template, Blackbaud staff should follow the new <a href="https://docs.blackbaud.com/stache-internal/new-bb-stache" title="Blackbaud staff users only">Create a New Stache SPA tutorial <i class="fa fa-lock" aria-hidden="true"></i></a> that we created just for you!
```
Comment (Blackbaud-DeniseCisneros) [inline src/app/support/changelog/2018-03/index.html]: To use the new template, Blackbaud staff should follow the new <a href="https://docs.blackbaud.com/stache-internal/new-bb-stache" title="Blackbaud staff users only">Create a New Stache SPA tutorial <i class="fa fa-lock" aria-hidden="true"></i></a>.

File: src/app/support/changelog/2018-03/index.html [hunk 6]
```diff
@@ -0,0 +1,39 @@
...
+- Added a tutorial for how to [migrate from Stache 1](/stache/learn/migrate).
+- Added a [Troubleshooting](/stache/support/troubleshoot) page to help you problem solve issues you may encounter.
+- Added external link icons <i class="fa fa-external-link" aria-hidden="true"></i>, to better warn you when you are leaving our website for another.
+- Added a locked icon <i class="fa fa-lock" aria-hidden="true"></i>, to show links that are restricted to Blackbaud staff only.
+- Updated the [home page](/stache/) to show Blackbaud staff specific benefits for you.
```
Comment (Blackbaud-DeniseCisneros) [inline src/app/support/changelog/2018-03/index.html]: Updated the [home page](/stache/) to show specific Blackbaud staff benefits.

File: src/app/support/changelog/2018-03/index.html [hunk 7]
```diff
@@ -0,0 +1,39 @@
...
+- Based on feedback, made improvements to the [Getting Started](/stache/learn/get-started) documentation. For Blackbaud staff, to better guide you for Stache SPAs, this also includes an improvement we made to the SKY UX App Management Portal. (Thank you, SKY UX Team!)
```
Comment (Blackbaud-DeniseCisneros) [inline src/app/support/changelog/2018-03/index.html]: Based on feedback, we made improvements to the [Getting Started](/stache/learn/get-started) documentation. To better guide Blackbaud staff with Stache SPAs, we also included an improvement to the SKY UX App Management Portal. (Thank you, SKY UX Team!)

File: src/app/support/changelog/2018-05/index.html [hunk 8]
```diff
@@ -0,0 +1,33 @@
...
+- Updated StacheRouterLink to include browser context menu options on right-click and shift/ctrl/cmd+click functionality for opening new tabs/windows. Users should now use StacheRouterLink instead of Angular RouterLink. [#458](https://github.com/blackbaud/stache2/pull/458)
+
+## 2.7.1 (2018-5-11)
+
+- Fixed an issue with the anchor icon on stache-page-anchor's not displaying. [#462](https://github.com/blackbaud/stache2/pull/462)
+
+## 2.7.0 (2018-5-9)
+
+- StacheRouterLink bugfixes [#452](https://github.com/blackbaud/stache2/pull/452)
+  - Fixed issues that stopped the page from navigating to a fragment on another page, if the current page also contained a fragment with the same name.
```
Comment (Blackbaud-DeniseCisneros) [inline src/app/support/changelog/2018-05/index.html]: Fixed issues that stopped the page from navigating to a fragment on another page when the current page contained a fragment with the same name.

File: src/app/support/changelog/2018-06/index.html [hunk 9]
```diff
@@ -0,0 +1,48 @@
...
+  </stache-page-summary>
+
+  <stache-markdown>
+## 2.9.0 (2018-06-28)
+- Style fixes. [#489](https://github.com/blackbaud/stache2/pull/489)
+  - The sidebar now properly positions itself under the omnibar, dev bar, and elements above the stache-layout.
+  - The sidebar now adjusts its height to not overlap the footer.
+  - The page layouts are now a max-width of 1600px.
+- Created the `stache-footer` component.
+  - Added to the footer component to the layout's.
```
Comment (Blackbaud-DeniseCisneros) [inline src/app/support/changelog/2018-06/index.html]: Added the footer component to layouts.

File: src/app/support/changelog/2018-06/index.html [hunk 10]
```diff
@@ -0,0 +1,48 @@
...
+
+  <stache-page-summary>
+    Monitor this page to keep up with the latest changes and releases for {{ stache.jsonData.global.productNameLong }}.
+  </stache-page-summary>
+
+  <stache-markdown>
+## 2.9.0 (2018-06-28)
+- Style fixes. [#489](https://github.com/blackbaud/stache2/pull/489)
+  - The sidebar now properly positions itself under the omnibar, dev bar, and elements above the stache-layout.
+  - The sidebar now adjusts its height to not overlap the footer.
```
Comment (Blackbaud-DeniseCisneros) [inline src/app/support/changelog/2018-06/index.html]: The sidebar now adjusts its height and does not overlap the footer.

File: src/app/support/changelog/2018-06/index.html [hunk 11]
```diff
@@ -0,0 +1,48 @@
...
+
+## 2.8.0 (2018-6-19)
+
+- Refactored and restyled the sidebar layout. [#475](https://github.com/blackbaud/stache2/pull/475)
+  - Created a `stache-sidebar-wrapper` component.
+  - Refactored the styles for the `stache-container` component.
+  - Removed use of the `sky-container` class.
+  - Made better use of the page real estate.
+  - Visually separated the nav from the page content as desired.
+  - Fixed a bug with jumping sidebar when switching between pages that contained a vertical scroll bar vrs no scroll bar.
```
Comment (Blackbaud-DeniseCisneros) [inline src/app/support/changelog/2018-06/index.html]: Fixed a bug with jumping sidebar when switching between pages that contained a vertical scroll bar vs. no scroll bar.

File: src/app/support/changelog/index.html [hunk 12]
```diff
@@ -1,25 +1,47 @@
...
-- Moved documentation (layouts, links, and navigation) from the Basics page to the [Content](/stache/content) page.
-- For Blackbaud staff, we created a new Stache SPA (stache-internal) to provide *internal to Blackbaud only* documentation. This enables us to provide very specific information for you, without "mucking up" documentation that's applicable to all users. As part of this, we created a centralized location for <a href="https://docs.blackbaud.com/stache-internal/search" title="Blackbaud staff only">Stache Search Service <i class="fa fa-lock" aria-hidden="true"></i></a> documentation. This means it's now available to find in the <a href="https://docs.blackbaud.com/stache-internal/search" title="Blackbaud staff only">global Stache search <i class="fa fa-lock" aria-hidden="true"></i></a>
-  </stache-markdown>
+## 2.9.0 (2018-06-28)
+- Style fixes. [#489](https://github.com/blackbaud/stache2/pull/489)
+  - The sidebar now properly positions itself under the omnibar, dev bar, and elements above the stache-layout.
+  - The sidebar now adjusts its height to not overlap the footer.
```
Comment (Blackbaud-DeniseCisneros) [inline src/app/support/changelog/index.html]: The sidebar now adjusts its height and does not overlap the footer.

File: src/app/support/changelog/index.html [hunk 13]
```diff
@@ -1,25 +1,47 @@
...
+
+## 2.8.0 (2018-6-19)
+
+- Refactored and restyled the sidebar layout. [#475](https://github.com/blackbaud/stache2/pull/475)
+  - Created a `stache-sidebar-wrapper` component.
+  - Refactored the styles for the `stache-container` component.
+  - Removed use of the `sky-container` class.
+  - Made better use of the page real estate.
+  - Visually separated the nav from the page content as desired.
+  - Fixed a bug with jumping sidebar when switching between pages that contained a vertical scroll bar vrs no scroll bar.
```
Comment (Blackbaud-DeniseCisneros) [inline src/app/support/changelog/index.html]: Fixed a bug with jumping sidebar when switching between pages that contained a vertical scroll bar vs. no scroll bar.

File: src/app/content/footer/index.html [hunk 14]
```diff
@@ -0,0 +1,76 @@
...
+  </stache-page-summary>
+
+  <h3>
+    Footer properties
+  </h3>
+
+  <ul>
+    <li>
+      <p>
+        <stache-code>copyrightLabel</stache-code> — This flag allows you to specify the additional copyright information after the current date. Defaults to <stache-code>Blackbaud, Inc. All rights reserved.</stache-code>
```
[the remark this answers is not in the document]
Comment (Blackbaud-BrandonJones) [reply src/app/content/footer/index.html]: 👍

File: src/app/support/changelog/2018-03/index.html (hunk 3 above)
Comment (Blackbaud-BrandonJones) [reply src/app/support/changelog/2018-03/index.html]: 👍

**Comment by Blackbaud-BrandonJones** [comment]
@Blackbaud-DeniseCisneros Updated per your feedback, thanks!

**Pull request merged**